### PR TITLE
searchbar fix

### PR DESCRIPTION
**Pull request by LarsS88**
Needed for search bar not to break when you have a script path

### CODE REVIEW

File: wsps/export.index [hunk 1]
```diff
@@ -83,3 +82,5 @@
     "828_ocspnavmenu": "828_OCSPNavMenu",
     "4_class_definitions": "4_Class-definitions"
 }
```
Comment (wgevaert) [inline wsps/export.index:84]: Are you sure that you need two `}` here?

**Review by wgevaert — CHANGES_REQUESTED** [review]
There seems to be some typo in the index file (Preferably you let pagesync generate the file by itself). and I am not sure if you should delete those widgets. Are you certain they are not used anywhere?

File: wsps/export.index (hunk 1 above)
Comment (LarsS88) [reply wsps/export.index:84]: I'm sure I don't. That must be an error in my merge. Weird, I thought for sure I discarded the lines above...
Anyway, it should be this:
`
    "4_class_definitions": "4_Class-definitions",
    "10_csp_searchbar": "10_Csp searchbar"
}
`

Comment (LarsS88) [reply wsps/export.index:84]: Ugh... with linebreaks...

**Comment by LarsS88** [comment]
> and I am not sure if you should delete those widgets. Are you certain they are not used anywhere?
The was replaced by the template here: https://gitlab.wikibase.nl/community/cspresources/-/commit/e2a55c9d281cdf1859aa96bc0be1403eb845042a

**Pull request merged**